csrc/device_lower/pass/alias_memory.cpp

```diff
@@ -942,6 +963,7 @@ class AllocationInfoMap : private kir::IrVisitor {
   void setAlias(AllocationInfo* from, AllocationInfo* to) {
     alias_map_[from] = to;
     from->alias_to = to->alloc_expr;
+    to->outer_aliased_by.push_back(from);
```

Comment:
What if B alias A, and C alias B, will `A.outer_aliased_by` has both `B` and `C`?

Reply:
Good point. Currently two hop aliases are not assigned but they could be in the future possible.

Reply:
I added an assertion here that `to->alias_to` is null.